# Generated by Django 5.2.5 on 2025-08-26 01:48

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('auth', '0012_alter_user_first_name_max_length'),
        ('usuarios', '0001_initial'),
    ]

    operations = [
        migrations.AddField(
            model_name='customuser',
            name='roles',
            field=models.ManyToManyField(blank=True, to='auth.group'),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='cedula',
            field=models.CharField(error_messages={'blank': 'La cédula es obligatoria.', 'null': 'La cédula no puede ser nula.', 'unique': 'Ya existe un usuario con esta cédula.'}, max_length=20, unique=True),
        ),
        migrations.AlterField(
            model_name='customuser',
            name='username',
            field=models.CharField(error_messages={'blank': 'Debes ingresar un nombre de usuario.', 'unique': 'Ya existe un usuario con ese nombre.'}, help_text='Requerido. Hasta 150 caracteres. Letras, números y @/./+/-/_ solamente.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='Nombre de usuario'),
        ),
    ]
